### ISSUE TITLE
Can't launch with minecraft

### ISSUE THREAD

**Issue by DarkShadow44** [GTNewHorizons/NewHorizonsCoreMod#36]
I cloned the master-branch and run "./gradlew runClient".
It then opened a window with the title "Select an mcp conf dir for the dobfuscator". Dunno what that is supposed to mean.

After removing CodeChickenCore, CodeChickenLib and NEI it starts - but complains about missing GalacticraftCore and MicdoodleCore.
After those are added too, it crashes again: "java.lang.NoSuchFieldError: field_78090_t"

In short, I can't get it to work - is there a tutorial on how to run this?

**Comment by Namikon** [comment]
You need the CodeChicken/Lib mods in order to run "non-dev" or "non-deobf" mods in the development environment. The MCP conf dir can usually be found at

> C:\\Users\[username]\\.gradle\\caches\\minecraft\\net\\minecraftforge\\forge\\1.7.10-10.xx.x.xxxx-1.7.10\\unpacked\\conf

The other mods need to go into the subfolder

> eclipse\mods

I don't have an up to date list of those; but it's at least the 2 you mentioned
Maybe @Dream-Master can provide a list

**Comment by DarkShadow44** [comment]
Ah thanks, the config did the tricks.
Btw, there is no eclipse folder for me, needed to set that up myself. And then create a launch configuration, and put the mods into NewHorizonsCoreMod/mods.

Maybe this could be automated or documented somewhere? My first idea was to download MCP since I couldn't find the "mcp conf dir", but that didn't work. Forge sources works fine, many thanks!

**Comment by Namikon** [comment]
Yeah, these are the things you have to do the first time you set the repository up on your dev machine, and then you're set. So there wasn't really a need to write a documentation for this yet

**Comment by DarkShadow44** [comment]
Well, when you're doing it the first time - like me just now - it can become a problem. I'm used to it just working when running "gradlew runClient".

**Comment by draknyte1** [comment]
This specifically doesn't need a guide because it's the same for every mod? You'd hit the same brick wall trying to build GT or GT++.
